FWCore/Integration/plugins/GlobalVectorAnalyzer.cc

```diff
@@ -0,0 +1,67 @@
+#include <vector>
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/global/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+template <typename T>
+T& operator<<(T& out, std::vector<double> const& values) {
```

Comment:
I'm wondering if it would be better to define this function template in an anonymous namespace to avoid any potential future ODR violations (thinking of another library defining similar `operator<<` for `cms::Exception` and `std::vector<double>`, and that library and this library ending up being loaded in the same (test) program).

Reply:
Sure.

Reply:
Done.